guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
+= Changing the Repository Sets Status in {Project}
+
+Repository sets show available repositories to each host.
+If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
+. Select the *Repository sets* tab.
+On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
+. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
+. On the *Overview* tab, click *Content view details* to view the environment for the host.
++
+[NOTE]
+====
+For hosts not in a default status or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.
```

Comment:
Re-commenting here so this doesn't get lost in the resolved comment. "Default status" isn't correct -- we use the word "default" here in two different ways. Really the meaning here is "for hosts not in the Default Organization View content view and Library lifecycle environment," but that's a bit cumbersome. We should also highlight that this is what makes the toggle group show. Maybe something like this:

```suggestion
For hosts not in the default content view and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, 'Limit to Environment' and 'Show all.'

Limit to environment: Show only repositories that are relevant to the host.
Show all: Show all available repositories, even those that may not be in the host's content view and lifecycle environment.
```
```suggestion
For hosts not in a default status or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.
```